src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.scss

```diff
@@ -73,3 +58,32 @@
 .controlButtons > * {
   flex-shrink: 0;
 }
+
+.deleteMessageContainer {
+  margin: 20px 0px;
```

Comment:
`0` doesn't need units. `0px` is the same as `0em` or `0rem` or `0ch`, or `0%`. It's just `0`.